lte/gateway/fabfile.py

```diff
@@ -149,7 +149,7 @@ def package(
                 'cat {}'.format(mirrored_packages_file)
                 + ' | xargs -I% sudo aptitude download -q2 %',
             )
-            run('cp *.deb ~/magma-packages')
+            run('cp *.deb ~/magma-packages 2>/dev/null')  # Ignore error if no packages have been built.
```

Comment:
does this only suppress the error message, but still returns >0? you could try `cp *.deb ~/magma-packages 2>/dev/null || :`

Reply:
The other question is why is it empty now and wasn't failing in this step before?

Reply:
I agree with @nstng.

@sebathomas I don't think we tried to run this workflow before with no dependencies to be built.

Reply:
Why? Wouldn't it be the default if there are no upgrades?

Reply:
Results of discussing in person:
- Before the Python upgrades, there were a few packages that were always built even though they already existed in artifactory. That's why we didn't run into this case before. This might be a bug in pydep.
- Actually, no packages being built if everything already exists should be okay.